src/ontology/foodon-edit.owl

```diff
@@ -41964,6 +42025,18 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/FOODON_03310576 -->
+
+    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/FOODON_03310576">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03302012"/>
+        <obo:IAO_0000115 xml:lang="en">A cooked and smoked sausage of chicken served in a bread roll.</obo:IAO_0000115>
```

Comment:
The term should have a full label "chicken frankfurter" and an exact synonym "chicken frank".  As well the definition only has to reference parent class, so "A frankfurter made entirely of chicken".

Reply:
This term is actually already present in FoodOn with a full label 'chicken frankfurter'. So it's more like an update, and that's why only added 'chicken frank' as an exact synonym. Agreed with the changes in definition, I'll do that. Thanks alot!

Reply:
you are so right about synonym!  Sorry, my bad!